lockfile/lockfile-file/src/sortLockfileKeys.ts

```diff
@@ -77,6 +77,9 @@ export function sortLockfileKeys (lockfile: LockfileFile) {
       })
     }
   }
+  if (lockfile.patchedDependencies) {
+    lockfile.patchedDependencies = sortKeys(lockfile.patchedDependencies)
+  }
   for (const key of ['specifiers', 'dependencies', 'devDependencies', 'optionalDependencies', 'time'] as const) {
```

Comment:
I tried just adding `'patchedDependencies'` on this line, but it led a complicated typescript error. I wasn't sure how to convince typescript it's ok without doing something unsafe like an `as any` cast